CodeEdit/NavigatorSidebar/FindNavigator/FindNavigatorSearchBar.swift

```diff
@@ -6,16 +6,62 @@
 //
 
 import SwiftUI
+import CodeEditUI
+
+extension Color {
+    init(hex: Int, opacity: Double = 1.0) {
+        let red = Double((hex & 0xff0000) >> 16) / 255.0
+        let green = Double((hex & 0xff00) >> 8) / 255.0
+        let blue = Double((hex & 0xff) >> 0) / 255.0
+        self.init(.sRGB, red: red, green: green, blue: blue, opacity: opacity)
+    }
+}
```

Comment:
Please extend the already available extensions in `CodeEditUtils/Extensions/Color+HEX.swift` to minimize code duplication.